components/calendar/calendar.js

```diff
@@ -395,6 +412,25 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 				font-size: 1rem;
 				font-weight: 700;
 			}
+
+			.d2l-calendar-date-day-info::after {
+				background-color: var(--d2l-color-celestine);
+				border-radius: 3px;
+				bottom: 4px;
+				content: "";
+				display: inline-block;
+				height: 6px;
+				left: calc(50% - 3px);
```

Comment:
Should this (and less importantly `bottom`) be using CSS logical properties?

Reply:
It looks like I can get rid of this entirely, since `d2l-calendar` justifies the content to be centered. The original `left` centered it though, so no flipped was originally required in MVC.